cmd/bosun/conf/actionNotify.go

```diff
@@ -0,0 +1,105 @@
+package conf
+
+import (
+	"bytes"
+	"fmt"
+	"net/url"
+	"strings"
+
+	"bosun.org/slog"
+
+	"bosun.org/cmd/bosun/conf/template"
+	"bosun.org/models"
+)
+
+var defaultActionNotificationSubjectTemplate *template.Template
```

Comment:
I believe you are generally calling these as a pair (i.e. `prepareFromTemplateKeys`), and the var name is crazy long. Maybe better to combine them into a struct like `defaultActionNotificationTemplates` with a Subject and Body property. That also makes it so the varying part of the variable name, `Subject` and `Template` are not in the middle of the var name.